### PR TITLE
docs: better documentation

### PR DESCRIPTION
**Pull request by joao-conde**
Issue: https://github.com/ripe-tech/ups-api-js/issues/27

Improves JSDocs of all methods and adds example usage to main README.

### CODE REVIEW

File: js/document.js [hunk 1]
```diff
@@ -17,6 +17,15 @@ export const DECLARATION_DOC_TYPE = "013";
 
 export const DocumentAPI = superclass =>
     class extends superclass {
+        /**
+         * Uploads a document to the UPS servers.
+         * The uploaded document is returned and its ID can be
+         * used to associate a shipment with the document.
+         *
+         * @param {Object} payload The payload object according to the UPS API standards.
```
Comment (BeeMargarida) [inline js/document.js:25]: Is there a link to a document/site that can be added here that would list these standards?

Comment (joao-conde) [reply js/document.js:25]: This one: https://www.ups.com/upsdeveloperkit?loc=en_US

Should it be @see https://www.ups.com/upsdeveloperkit?loc=en_US ?

Comment (BeeMargarida) [reply js/document.js:25]: Yap!

Comment (joao-conde) [reply js/document.js:25]: Sadly cant get an individual link for each API section. So I will use the generic one for all.

**Review by BeeMargarida — APPROVED** [review]

**Review by 3rdvision — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**